sites/cheerpx/src/content/docs/00-overview.mdx

```diff
@@ -28,9 +68,36 @@ CheerpX enables you to run existing 32-bit x86 native binaries and Linux distrib
 - Without source-code
 - Fully client-side
 
-CheerpX is perfect for web-based IDEs, programming tutorials, virtual machines, and interactive demos. It can also be used to provide a secure sandbox for untrusted code. CheerpX is very robust and deployed as part of [WebVM](https://webvm.io/) and [PythonFiddle](https://pythonfiddle.leaningtech.com/). It is also used in production environments as part of [CheerpX for Flash](https://labs.leaningtech.com/docs/cheerpx-for-flash), an enterprise solution to run legacy Adobe Flash content in the browser.
+It can also be used to provide a secure sandbox for untrusted code. CheerpX is very robust and deployed as part of [WebVM](https://webvm.io/) and [PythonFiddle](https://pythonfiddle.leaningtech.com/). It is also used in production environments as part of [CheerpX for Flash](https://labs.leaningtech.com/docs/cheerpx-for-flash), an enterprise solution to run legacy Adobe Flash content in the browser.
+
+<div class="text-stone-400">
+	If you're looking to run Java in the browser, please check out our specialized
+	solution: [CheerpJ](https://cheerpj.com/).
+</div>
+
+<LinkButton label="Get started" type="primary" href="/docs/getting-started" />
+
+## Licensing
+
+CheerpX is free to use for personal projects, most non-commercial applications, and for technical evaluations. See [licensing](/docs/licensing) for details.
 
-If you're looking to run Java in the browser, please check out our specialized solution: [CheerpJ](https://cheerpj.com/).
+## How does CheerpX work?
+
+At its core, CheerpX is a two-tier emulator for the x86 architecture [implemented in C++ and compiled to JavaScript and WebAssembly](https://cheerp.io/). There is both an interpreter and a sophisticated JIT compiler that is able to generate efficient WebAssembly representations for hot code. The whole process is 100% automated and requires no metadata or preprocessing steps. The system is so robust that it can deal with applications that generate new code internally at runtime, and even with self-modifying code.
```

Comment:
```suggestion
At its core, CheerpX is a two-tier emulator for the x86 architecture implemented in C++ and [compiled to JavaScript and WebAssembly](https://cheerp.io/). There is both an interpreter and a sophisticated JIT compiler that is able to generate efficient WebAssembly representations for hot code. The whole process is 100% automated and requires no metadata or preprocessing steps. The system is so robust that it can deal with applications that generate new code internally at runtime, and even with self-modifying code.
```
Link is too long